ClothesApp/Infrastructure/Repositories/OrderTransactionRepository.cs

```diff
@@ -0,0 +1,28 @@
+using Application.Interfaces.Repositories;
+using Domain.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace Infrastructure.Repositories;
+
+public class OrderTransactionRepository : IOrderTransactionsRepository
+{
+    private readonly ClothesAppContext _dbContext;
+
+    public OrderTransactionRepository(ClothesAppContext dbContext)
+    {
+        _dbContext = dbContext;
+    }
+
+    public async Task<OrderTransaction> Add(OrderTransaction orderTransaction)
```

Comment:
Why naming is different compared to generic repo

Reply:
And why did you decide to stop using your generic repo interface

Reply:
It just looks like your generic repository is not working, you have multiple notimplementedexceptions and at least one repository that doesn't even use this interface. Anyway it's good experience and there is no need to update it now